### ISSUE TITLE
Using almost every conda command results in similair error response. (ModuleNotFoundError: No module named 'chardet')

### ISSUE THREAD

**Issue by Bleeplo** [conda/conda#12692]
### Checklist

- [X] I added a descriptive title
- [X] I searched open reports and couldn't find a duplicate

### What happened?

I am on Linux (Manjaro KDE, specifically), and I have recently installed minconda via the [conda website](https://docs.conda.io/en/latest/miniconda.html). Every-time I try running a command, such as create or install, I get a error saying:
```
# >>>>>>>>>>>>>>>>>>>>>> ERROR REPORT <<<<<<<<<<<<<<<<<<<<<<

    Traceback (most recent call last):
      File "/home/phil/miniconda3/lib/python3.10/site-packages/conda/exceptions.py", line 1132, in __call__
        return func(*args, **kwargs)
      File "/home/phil/miniconda3/lib/python3.10/site-packages/conda/cli/main.py", line 69, in main_subshell
        exit_code = do_call(args, p)
      File "/home/phil/miniconda3/lib/python3.10/site-packages/conda/cli/conda_argparse.py", line 120, in do_call
        module = import_module(relative_mod, __name__.rsplit('.', 1)[0])
      File "/home/phil/miniconda3/lib/python3.10/importlib/__init__.py", line 126, in import_module
        return _bootstrap._gcd_import(name[level:], package, level)
      File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
      File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
      File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
      File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
      File "<frozen importlib._bootstrap_external>", line 883, in exec_module
      File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
      File "/home/phil/miniconda3/lib/python3.10/site-packages/conda/cli/main_create.py", line 14, in <module>
        from ..notices import notices
      File "/home/phil/miniconda3/lib/python3.10/site-packages/conda/notices/__init__.py", line 3, in <module>
        from .core import notices  # noqa: F401
      File "/home/phil/miniconda3/lib/python3.10/site-packages/conda/notices/core.py", line 16, in <module>
        from . import fetch
      File "/home/phil/miniconda3/lib/python3.10/site-packages/conda/notices/fetch.py", line 8, in <module>
        import requests
      File "/home/phil/.local/lib/python3.10/site-packages/requests/__init__.py", line 44, in <module>
        import chardet
    ModuleNotFoundError: No module named 'chardet'

`$ /home/phil/miniconda3/bin/conda create -n textgen python=3.10.9`
```
However, I checked to see if I had chardet installed, via running the command ` pip install chardet`. Which results in:
```
Defaulting to user installation because normal site-packages is not writeable
Requirement already satisfied: chardet in /usr/lib/python3.10/site-packages (5.0.0)
```
If I am understanding correctly, I have chardet installed already. So, I am perplexed as to why I am receiving such an error. Also! Every-time I run **any** conda command, I receive this at the top of every command response:
```
/home/phil/.local/lib/python3.10/site-packages/_distutils_hack/__init__.py:33: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
```

### Conda Info

```shell
(base) [phil@phil-manjarolinux ~]$ conda info
/home/phil/.local/lib/python3.10/site-packages/_distutils_hack/__init__.py:33: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
/home/phil/.local/lib/python3.10/site-packages/_distutils_hack/__init__.py:33: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")

     active environment : base
    active env location : /home/phil/miniconda3
            shell level : 1
       user config file : /home/phil/.condarc
 populated config files : 
          conda version : 23.3.1
    conda-build version : not installed
         python version : 3.10.10.final.0
       virtual packages : __archspec=1=x86_64
                          __cuda=11.8=0
                          __glibc=2.36=0
                          __linux=6.0.8=0
                          __unix=0=0
       base environment : /home/phil/miniconda3  (writable)
      conda av data dir : /home/phil/miniconda3/etc/conda
  conda av metadata url : None
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/phil/miniconda3/pkgs
                          /home/phil/.conda/pkgs
       envs directories : /home/phil/miniconda3/envs
                          /home/phil/.conda/envs
               platform : linux-64
             user-agent : conda/23.3.1 requests/2.28.2 CPython/3.10.10 Linux/6.0.8-1-MANJARO manjaro/22.0.0 glibc/2.36
                UID:GID : 1000:1000
             netrc file : None
           offline mode : False
```

### Conda Config

```shell
/home/phil/.local/lib/python3.10/site-packages/_distutils_hack/__init__.py:33: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
```

### Conda list

```shell
/home/phil/.local/lib/python3.10/site-packages/_distutils_hack/__init__.py:33: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
# packages in environment at /home/phil/miniconda3:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main    defaults
_openmp_mutex             5.1                       1_gnu    defaults
boltons                   23.0.0          py310h06a4308_0    defaults
brotlipy                  0.7.0           py310h7f8727e_1002    defaults
bzip2                     1.0.8                h7b6447c_0    defaults
ca-certificates           2023.01.10           h06a4308_0    defaults
certifi                   2022.12.7       py310h06a4308_0    defaults
cffi                      1.15.1          py310h5eee18b_3    defaults
charset-normalizer        2.0.4              pyhd3eb1b0_0    defaults
conda                     23.3.1          py310h06a4308_0    defaults
conda-content-trust       0.1.3           py310h06a4308_0    defaults
conda-package-handling    2.0.2           py310h06a4308_0    defaults
conda-package-streaming   0.7.0           py310h06a4308_0    defaults
cryptography              39.0.1          py310h9ce1e76_0    defaults
idna                      3.4             py310h06a4308_0    defaults
jsonpatch                 1.32               pyhd3eb1b0_0    defaults
jsonpointer               2.1                pyhd3eb1b0_0    defaults
ld_impl_linux-64          2.38                 h1181459_1    defaults
libffi                    3.4.2                h6a678d5_6    defaults
libgcc-ng                 11.2.0               h1234567_1    defaults
libgomp                   11.2.0               h1234567_1    defaults
libstdcxx-ng              11.2.0               h1234567_1    defaults
libuuid                   1.41.5               h5eee18b_0    defaults
ncurses                   6.4                  h6a678d5_0    defaults
openssl                   1.1.1t               h7f8727e_0    defaults
packaging                 23.0            py310h06a4308_0    defaults
pip                       23.0.1          py310h06a4308_0    defaults
pluggy                    1.0.0           py310h06a4308_1    defaults
pycosat                   0.6.4           py310h5eee18b_0    defaults
pycparser                 2.21               pyhd3eb1b0_0    defaults
pyopenssl                 23.0.0          py310h06a4308_0    defaults
pysocks                   1.7.1           py310h06a4308_0    defaults
python                    3.10.10              h7a1cb2a_2    defaults
readline                  8.2                  h5eee18b_0    defaults
requests                  2.28.1          py310h06a4308_1    defaults
ruamel.yaml               0.17.21         py310h5eee18b_0    defaults
ruamel.yaml.clib          0.2.6           py310h5eee18b_1    defaults
setuptools                65.6.3          py310h06a4308_0    defaults
six                       1.16.0             pyhd3eb1b0_1    defaults
sqlite                    3.41.1               h5eee18b_0    defaults
tk                        8.6.12               h1ccaba5_0    defaults
toolz                     0.12.0          py310h06a4308_0    defaults
tqdm                      4.65.0          py310h2f386ee_0    defaults
tzdata                    2023c                h04d1e81_0    defaults
urllib3                   1.26.15         py310h06a4308_0    defaults
wheel                     0.38.4          py310h06a4308_0    defaults
xz                        5.2.10               h5eee18b_1    defaults
zlib                      1.2.13               h5eee18b_0    defaults
zstandard                 0.19.0          py310h5eee18b_0    defaults
```

### Additional Context

I know I am using conda 23.3.1, because when I run the command `conda --version`, it replies with:
```
(base) [phil@phil-manjarolinux ~]$ conda --version
/home/phil/.local/lib/python3.10/site-packages/_distutils_hack/__init__.py:33: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
conda 23.3.1

```

**Comment by Bleeplo** [comment]
Running command `conda update`:
```
/home/phil/.local/lib/python3.10/site-packages/_distutils_hack/__init__.py:33: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
/home/phil/.local/lib/python3.10/site-packages/_distutils_hack/__init__.py:33: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")

# >>>>>>>>>>>>>>>>>>>>>> ERROR REPORT <<<<<<<<<<<<<<<<<<<<<<

    Traceback (most recent call last):
      File "/home/phil/miniconda3/lib/python3.10/site-packages/conda/exceptions.py", line 1132, in __call__
        return func(*args, **kwargs)
      File "/home/phil/miniconda3/lib/python3.10/site-packages/conda/cli/main.py", line 69, in main_subshell
        exit_code = do_call(args, p)
      File "/home/phil/miniconda3/lib/python3.10/site-packages/conda/cli/conda_argparse.py", line 120, in do_call
        module = import_module(relative_mod, __name__.rsplit('.', 1)[0])
      File "/home/phil/miniconda3/lib/python3.10/importlib/__init__.py", line 126, in import_module
        return _bootstrap._gcd_import(name[level:], package, level)
      File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
      File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
      File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
      File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
      File "<frozen importlib._bootstrap_external>", line 883, in exec_module
      File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
      File "/home/phil/miniconda3/lib/python3.10/site-packages/conda/cli/main_update.py", line 8, in <module>
        from ..notices import notices
      File "/home/phil/miniconda3/lib/python3.10/site-packages/conda/notices/__init__.py", line 3, in <module>
        from .core import notices  # noqa: F401
      File "/home/phil/miniconda3/lib/python3.10/site-packages/conda/notices/core.py", line 16, in <module>
        from . import fetch
      File "/home/phil/miniconda3/lib/python3.10/site-packages/conda/notices/fetch.py", line 8, in <module>
        import requests
      File "/home/phil/.local/lib/python3.10/site-packages/requests/__init__.py", line 44, in <module>
        import chardet
    ModuleNotFoundError: No module named 'chardet'

`$ /home/phil/miniconda3/bin/conda update`

  environment variables:
                 CIO_TEST=<not set>
        CONDA_DEFAULT_ENV=base
                CONDA_EXE=/home/phil/miniconda3/bin/conda
             CONDA_PREFIX=/home/phil/miniconda3
    CONDA_PROMPT_MODIFIER=(base)
         CONDA_PYTHON_EXE=/home/phil/miniconda3/bin/python
               CONDA_ROOT=/home/phil/miniconda3
              CONDA_SHLVL=1
           CURL_CA_BUNDLE=<not set>
               LD_PRELOAD=<not set>
                     PATH=/home/phil/miniconda3/bin:/home/phil/miniconda3/condabin:/home/phil/.l
                          ocal/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/home/phil/.dotnet/to
                          ols:/var/lib/flatpak/exports/bin:/usr/lib/jvm/default/bin:/usr/bin/sit
                          e_perl:/usr/bin/vendor_perl:/usr/bin/core_perl:/var/lib/snapd/snap/bin
       REQUESTS_CA_BUNDLE=<not set>
            SSL_CERT_FILE=<not set>
            XDG_SEAT_PATH=/org/freedesktop/DisplayManager/Seat0
         XDG_SESSION_PATH=/org/freedesktop/DisplayManager/Session2

     active environment : base
    active env location : /home/phil/miniconda3
            shell level : 1
       user config file : /home/phil/.condarc
 populated config files : 
          conda version : 23.3.1
    conda-build version : not installed
         python version : 3.10.10.final.0
       virtual packages : __archspec=1=x86_64
                          __cuda=11.8=0
                          __glibc=2.36=0
                          __linux=6.0.8=0
                          __unix=0=0
       base environment : /home/phil/miniconda3  (writable)
      conda av data dir : /home/phil/miniconda3/etc/conda
  conda av metadata url : None
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/phil/miniconda3/pkgs
                          /home/phil/.conda/pkgs
       envs directories : /home/phil/miniconda3/envs
                          /home/phil/.conda/envs
               platform : linux-64
             user-agent : conda/23.3.1 requests/2.28.2 CPython/3.10.10 Linux/6.0.8-1-MANJARO manjaro/22.0.0 glibc/2.36
                UID:GID : 1000:1000
             netrc file : None
           offline mode : False


An unexpected error has occurred. Conda has prepared the above report.

If submitted, this report will be used by core maintainers to improve
future releases of conda.
Would you like conda to send this report to the core maintainers? [y/N]: y
Upload did not complete.

Thank you for helping to improve conda.
Opt-in to always sending reports (and not see this message again)
by running

    $ conda config --set report_errors true
```
And, running an install command. For example, `conda install nimble`:
```
/home/phil/.local/lib/python3.10/site-packages/_distutils_hack/__init__.py:33: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")
/home/phil/.local/lib/python3.10/site-packages/_distutils_hack/__init__.py:33: UserWarning: Setuptools is replacing distutils.
  warnings.warn("Setuptools is replacing distutils.")

# >>>>>>>>>>>>>>>>>>>>>> ERROR REPORT <<<<<<<<<<<<<<<<<<<<<<

    Traceback (most recent call last):
      File "/home/phil/miniconda3/lib/python3.10/site-packages/conda/exceptions.py", line 1132, in __call__
        return func(*args, **kwargs)
      File "/home/phil/miniconda3/lib/python3.10/site-packages/conda/cli/main.py", line 69, in main_subshell
        exit_code = do_call(args, p)
      File "/home/phil/miniconda3/lib/python3.10/site-packages/conda/cli/conda_argparse.py", line 120, in do_call
        module = import_module(relative_mod, __name__.rsplit('.', 1)[0])
      File "/home/phil/miniconda3/lib/python3.10/importlib/__init__.py", line 126, in import_module
        return _bootstrap._gcd_import(name[level:], package, level)
      File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
      File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
      File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
      File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
      File "<frozen importlib._bootstrap_external>", line 883, in exec_module
      File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
      File "/home/phil/miniconda3/lib/python3.10/site-packages/conda/cli/main_install.py", line 8, in <module>
        from ..notices import notices
      File "/home/phil/miniconda3/lib/python3.10/site-packages/conda/notices/__init__.py", line 3, in <module>
        from .core import notices  # noqa: F401
      File "/home/phil/miniconda3/lib/python3.10/site-packages/conda/notices/core.py", line 16, in <module>
        from . import fetch
      File "/home/phil/miniconda3/lib/python3.10/site-packages/conda/notices/fetch.py", line 8, in <module>
        import requests
      File "/home/phil/.local/lib/python3.10/site-packages/requests/__init__.py", line 44, in <module>
        import chardet
    ModuleNotFoundError: No module named 'chardet'

`$ /home/phil/miniconda3/bin/conda install nimble`

  environment variables:
                 CIO_TEST=<not set>
        CONDA_DEFAULT_ENV=base
                CONDA_EXE=/home/phil/miniconda3/bin/conda
             CONDA_PREFIX=/home/phil/miniconda3
    CONDA_PROMPT_MODIFIER=(base)
         CONDA_PYTHON_EXE=/home/phil/miniconda3/bin/python
               CONDA_ROOT=/home/phil/miniconda3
              CONDA_SHLVL=1
           CURL_CA_BUNDLE=<not set>
               LD_PRELOAD=<not set>
                     PATH=/home/phil/miniconda3/bin:/home/phil/miniconda3/condabin:/home/phil/.l
                          ocal/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/home/phil/.dotnet/to
                          ols:/var/lib/flatpak/exports/bin:/usr/lib/jvm/default/bin:/usr/bin/sit
                          e_perl:/usr/bin/vendor_perl:/usr/bin/core_perl:/var/lib/snapd/snap/bin
       REQUESTS_CA_BUNDLE=<not set>
            SSL_CERT_FILE=<not set>
            XDG_SEAT_PATH=/org/freedesktop/DisplayManager/Seat0
         XDG_SESSION_PATH=/org/freedesktop/DisplayManager/Session2

     active environment : base
    active env location : /home/phil/miniconda3
            shell level : 1
       user config file : /home/phil/.condarc
 populated config files : 
          conda version : 23.3.1
    conda-build version : not installed
         python version : 3.10.10.final.0
       virtual packages : __archspec=1=x86_64
                          __cuda=11.8=0
                          __glibc=2.36=0
                          __linux=6.0.8=0
                          __unix=0=0
       base environment : /home/phil/miniconda3  (writable)
      conda av data dir : /home/phil/miniconda3/etc/conda
  conda av metadata url : None
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /home/phil/miniconda3/pkgs
                          /home/phil/.conda/pkgs
       envs directories : /home/phil/miniconda3/envs
                          /home/phil/.conda/envs
               platform : linux-64
             user-agent : conda/23.3.1 requests/2.28.2 CPython/3.10.10 Linux/6.0.8-1-MANJARO manjaro/22.0.0 glibc/2.36
                UID:GID : 1000:1000
             netrc file : None
           offline mode : False


An unexpected error has occurred. Conda has prepared the above report.

If submitted, this report will be used by core maintainers to improve
future releases of conda.
Would you like conda to send this report to the core maintainers? [y/N]: y
Upload did not complete.

Thank you for helping to improve conda.
Opt-in to always sending reports (and not see this message again)
by running

    $ conda config --set report_errors true
```

**Comment by Bleeplo** [comment]
I solved the issue. I am perplexed to why now it decided to work. I know in my first write-up, I mentioned how I tried running the command `pip install chardet`. I did that in zsh, instead of bash. I don't know if that made a difference. As when I put my terminal in 'bash mode', and ran the command `pip install chardet`. It decided to actually install `chardet` this time, allowing me to use commands (for now).

However, I still get a extra warning when any command.
```
/home/phil/.local/lib/python3.10/site-packages/requests/__init__.py:87: RequestsDependencyWarning: urllib3 (1.26.15) or chardet (5.1.0) doesn't match a supported version!
  warnings.warn("urllib3 ({}) or chardet ({}) doesn't match a supported "
```

**TL;DR**: Run terminal in bash, paste `pip install chardet`

**Comment by flameleo11** [comment]
1. Issue: Dependency Warning for urllib3 or chardet not matching supported versions.
   Solution: Uninstall urllib3 and chardet, then upgrade the requests module.
   Steps:
   - `pip uninstall urllib3`
   - `pip uninstall chardet`
   - `pip install --upgrade requests`

2. Issue: RequestsDependencyWarning for incompatible urllib3 and chardet versions.
   Solution: Update urllib3 and chardet to match supported versions.
   Steps:
   - `pip install --upgrade urllib3`
   - `pip install --upgrade chardet`

**Comment by heena60034117** [comment]
> I solved the issue. I am perplexed to why now it decided to work. I know in my first write-up, I mentioned how I tried running the command `pip install chardet`. I did that in zsh, instead of bash. I don't know if that made a difference. As when I put my terminal in 'bash mode', and ran the command `pip install chardet`. It decided to actually install `chardet` this time, allowing me to use commands (for now).
> 
> However, I still get a extra warning when any command.
> 
> ```
> /home/phil/.local/lib/python3.10/site-packages/requests/__init__.py:87: RequestsDependencyWarning: urllib3 (1.26.15) or chardet (5.1.0) doesn't match a supported version!
>   warnings.warn("urllib3 ({}) or chardet ({}) doesn't match a supported "
> ```
> 
> **TL;DR**: Run terminal in bash, paste `pip install chardet`

Yes, it works for me too. Thank you :-)
